### PR TITLE
Update changelog with info about new classes vs old classes lookup-in…

### PR DESCRIPTION
**Pull request by freider**
…compatiblity

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ We appreciate your patience while we speedily work towards a stable release of t
 Previously each `@method` and web endpoint of a class would get its own set of isolated containers and never run in the same container as other sibling methods. 
 Starting in this version, all `@methods` and web endpoints will be part of the same container pool. Notably, this means all methods will scale up/down together, and options like `keep_warm` and `concurrency_limit` will affect the total number of containers for all methods in the class combined, rather than individually.
 
+**Version incompatibility:** Older client's (<0.63) can't use classes deployed by new clients (>=0.63) and vice versa. Apps or standalone clients using `Cls.lookup(...)` to invoke Modal classes need to be upgraded to version `>0.63` at the same time as the deployed app that's being called into.
```
Comment (mwaskom) [inline CHANGELOG.md:27]:
```suggestion
**Version incompatibility warning:** Older clients (<0.63) can't use classes deployed by new clients (>=0.63) and vice versa. Apps or standalone clients using `Cls.lookup(...)` to invoke Modal classes need to be upgraded to version `>=0.63` at the same time as the deployed app that's being called into.       

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ We appreciate your patience while we speedily work towards a stable release of t
 
 ### 0.63.2 (2024-06-25)
 
-* Fixes issue with `Cls.lookup` not working after upgrading to v0.63.0
+* Fixes issue with `Cls.lookup` not working (at all) after upgrading to v0.63.0. **Note**: this doesn't fix the cross-version from 0.63.0.
```
Comment (mwaskom) [inline CHANGELOG.md:15]:
```suggestion
* Fixes issue with `Cls.lookup` not working (at all) after upgrading to v0.63.0. **Note**: this doesn't fix the cross-version lookup incompatibility introduced in 0.63.0.
```

**Review by mwaskom — APPROVED** [review]
:+1: two grammar suggestions but otherwise lg, thanks for doing this.

**Comment by freider** [comment]
> 👍 two grammar suggestions but otherwise lg, thanks for doing this.

Thanks for catching!

**Pull request merged**